doc/code/__init__.rst

```diff
@@ -19,3 +19,11 @@ Module: catalyst.debug
     :no-heading:
     :no-inheritance-diagram:
     :fullname:
+
+Module: catalyst.cuda

```

Comment:
@josh146 Would you like to add the `cuda-quantum` integration to the overview page as well?

Reply:
Nope, think it's fine for now (I added it to the PL docs instead)